### PR TITLE
use ExtractorBase for Swift

### PR DESCRIPTION
**Pull request by asurkov**

### CODE REVIEW

File: src/main/kotlin/app/extractors/SwiftExtractor.kt [hunk 1]
```diff
@@ -4,35 +4,7 @@
...
-        return super.tokenize(newLine)
-    }
-
-    override fun getLanguageName(): String? {
-        return LANGUAGE_NAME
-    }
-}
+object SwiftExtractor : ExtractorBase(
+    language = Lang.SWIFT,
+    importRegex = Regex("""^(.*import)\s[^\n]*"""),
```
Comment (anatolystansler) [inline src/main/kotlin/app/extractors/SwiftExtractor.kt]: `extractImportRegex` used for actually extracting of imports for choosing right classifiers
`importRegex` used to filter tokens from imports and as @yaronskaya works slightly faster

Here importRegex (less accurate version) used for both extracting and filter tokens. @yaronskaya review this change.

Comment (yaronskaya) [reply src/main/kotlin/app/extractors/SwiftExtractor.kt]: @asurkov this regex doesn't extract imports. Please, use 'extractImportRegex' from previous version of SwiftExtractor. I wonder if it passes tests:)

**Comment by asurkov** [comment]
@yaronskaya now I see what you say. The tests indeed pass, which means it has to be a problem with the tests. For example, there's one `val line = "import Foundation"`, which is supposed to extract no libraries. Do you know whether this is correct one? Is there any special about Swift import instructions?

**Comment by yaronskaya** [comment]
@asurkov it turned out that we don't have tests of imports for Swift, there are only tests for libraries extraction. Could you please add them?

File: src/test/kotlin/test/tests/extractors/ExtractorTest.kt [hunk 2]
```diff
@@ -368,4 +357,33 @@ class ExtractorTest : Spek({
...
+            var lines = listOf("/* class City: RLMObject { */")
+            var actualLineImports = SwiftExtractor.extractImports(lines)
+            assertTrue(actualLineImports.isEmpty())
+        }
+        it("Swift imports") {
+            var lines = listOf(
+                "import UIKit"
+            )
+            var actualLineImports = SwiftExtractor.extractImports(lines)
+            assertEquals(actualLineImports, listOf("UIKit"))
```
Comment (yaronskaya) [inline src/test/kotlin/test/tests/extractors/ExtractorTest.kt]: Use `assertExtractsImport`

**Pull request merged**